### ISSUE TITLE
No block for metalloids, nonmetals, particular radioactive elements?

### ISSUE THREAD

**Issue by Noobly-Walker** [al132mc/chem-lib#55]
It appears as though there exists no block for Boron, Silicon, Phosphorus, Sulphur, Germanium, Arsenic, Selenium, Technetium, Antimony, Tellurium, Iodine, Promethium, Astatine, Neptunium, and Plutonium. There exists a powder for many of these elements, and indeed these are all solid at STP. Most of the excluded radioactive elements are even full metals. But the lack of blocks for these elements seems arbitrary.
The metalloids and nonmetals may not make complete sense to turn into ingots, but they can still be turned into larger crystals by melting them down. I have with me, in real life, a large silicon crystal, just a bit smaller than my fist. From there, it's not too much of a stretch to turn them into blocks. Even without the crystal as an intermediate, the precedent for powder directly into blocks lays with redstone and glowstone.
It appears as though the limitation for radioactive materials was "whatever doesn't appear in Earth's crust", but that doesn't explain the francium block. If this is indeed the case, it must be stated that there are but a few atoms of francium on Earth at any given time, and natural neptunium and plutonium is actually a fair bit more common than that. Technetium, Promethium, and Astatine can similarly be found in trace amounts here and there.
Given that the mod often paired with this allows obtaining of heavier elements through particle accelerators, I recommend adding ingots and blocks for elements up to Einsteinium or so, given that we have been able to produce visible amounts of them. Arguably, even further, as while we have yet to produce visible amounts of such things as Dubnium, it should theoretically be possible to do so.

**TL;DR:** Add blocks for Boron, Silicon, Phosphorus, Sulphur, Germanium, Arsenic, Selenium, Technetium, Antimony, Tellurium, Iodine, Promethium, and Astatine. Also add blocks and dust for Neptunium and Plutonium, as well as possibly Americium, Curium, Berkelium, Californium, and Einsteinium, and VERY possibly dust for every element up to Dubnium.
